### ISSUE TITLE
Ideas for content? Share them here.

### ISSUE THREAD

**Issue by unshakenme** [unshakenme/OSPAT#5]
As a starting point for collecting new curriculum content ideas, please share any resources as a comment below.

**Comment by unshakenme** [comment]
Suggestion from Twitter: Chris Letheby, Philosophy of Psychedelics

**Comment by unshakenme** [comment]
Update with these sections…
Self-awareness/ personality tests:

- https://www.psychologistworld.com/tests/jung-archetype-quiz
- https://openpsychometrics.org/
- https://www.psychologytoday.com/us/tests/relationships/relationship-attachment-style-test

Indigenous knowledge & Cultural considerations:
- https://akjournals.com/configurable/content/journals$002f2054$002f4$002f1$002farticle-p16.xml?t:ac=journals%24002f2054%24002f4%24002f1%24002farticle-p16.xml
- https://akjournals.com/view/journals/2054/4/1/article-p4.xml
- https://chacruna.net/mushrooms-resistance-sierra-mazateca/
- https://chacruna.net/sacred-mushrooms-of-the-mazatec-tradition-transforming-the-inner-landscape-of-the-human-psyche/
- https://chacruna.net/ayahuasca-cultural-heritage-brazil/
- https://chacruna.net/native-american-statement-regarding-decriminalization-of-peyote/
- https://chacruna.net/collective-accountability-plant-medicine/
- https://chacruna.net/colonial-shadows-in-the-psychedelic-renaissance/
- https://chacruna.net/considerations-for-psychedelic-therapists-when-working-with-nave-american-people-and-communities/
- https://chacruna.net/appropriation-misuse-ancestral-yage/
- https://chacruna.net/researchers_demand_reparations_mazatecs/
- https://chacruna.net/10-calls-to-action-toward-an-lgbtq-affirmative-psychedelic-therapy/

5D ASC Scale:
- https://www.researchgate.net/file.PostFileLoader.html?id=562e5e0b6307d9997c8b45a4&assetKey=AS:288852905349121@1445879307530&__cf_chl_tk=SIg1OvNQjS9AtESEna3KlMfNjeloZCPS4Uatu8s3Vyo-1647033036-0-gaNycGzNF70

Research:
- https://link.springer.com/article/10.1007/s00213-020-05710-w?wt_mc=Internal.Event.1.SEM.ArticleAuthorOnlineFirst&utm_source=ArticleAuthorOnlineFirst&utm_medium=email&utm_content=AA_en_06082018&ArticleAuthorOnlineFirst_20201122&error=cookies_not_supported&error=cookies_not_supported&code=229e0e65-3a5c-42f0-a024-528b3d37923b&code=82fea945-4af7-436d-9281-32ec31b969c3
- https://pubmed.ncbi.nlm.nih.gov/27909165/

Clinical Skills/awareness:
- https://youtu.be/NvZi7ZV-SWI
- https://www.goodtherapy.org/blog/power-differential-why-it-matters-so-much-in-therapy-1009154
- https://2017.psychedelicscience.org/conference/interdisciplinary/exploring-psychoanalytic-themes-and-therapeutic-alliance-in-mdma-assisted-psychotherapy
- https://enhancedtherapy.ca/inside-the-maps-mdma-assisted-therapy-training-program-with-shannon-carlin-enhanced-therapy-podcast-7/?mc_cid=9581417be5&mc_eid=f095d8edd2
- https://youtu.be/La9qnPwx3Nc
-

**Comment by CheekeyMonkey** [comment]
Really like the idea of ecodelics, i.e. that psychedelics as a tool can reset the human mind back to default ecocentricism opposed to modern egocentricism. Therse two papers are a fantastic reference... 

1. Lifetime experience with (classic) psychedelics predicts pro-environmental behavior through an increase in nature relatedness. Matthias Forstmann and Christina Sagioglou Journal of Psychopharmacology 2017. DOI: 10.1177/0269881117714049

2. From Egoism to Ecoism: Psychedelics Increase Nature Relatedness in a State-Mediated and Context-Dependent Manner. Hannes Kettner, Sam Gandy, Eline C. H. M. Haijen and Robin L. Carhart-Harris Centre for Psychedelic Research, Department of Brain Sciences, Faculty of Medicine, Imperial College London. International Journal of Environmental Research and Public Health. Published: December 2019 DOI: 10.3390/ijerph16245147

**Comment by unshakenme** [comment]
Agreed that would be another great addition to the "elective" section.

**Comment by voyagermedical** [comment]
Perhaps inclusion of The entropic brain: a theory of conscious states informed by neuroimaging research with psychedelic drugs? i.e. https://www.frontiersin.org/articles/10.3389/fnhum.2014.00020/full

**Comment by unshakenme** [comment]
Yes this would be another good one to add.

**Comment by Corinaromo** [comment]
:)
